projects/uitdatabank/reference/entry.json

```
@@ -3272,6 +3272,115 @@
         }
       }
     },
+    "/images": {
+      "post": {
+        "summary": "image - create",
```

```suggestion
        "summary": "image - upload",
```

I have a feeling that this would align more with what an integrator would look for when trying to find how to upload an image to our API / server

---

Yeah, I was doubting between the two, went for the consistency option. This makes more sense though 👍 .